formula-rxjava3/src/main/java/com/instacart/formula/rxjava3/RxJavaRuntimeExtensions.kt

```diff
@@ -40,7 +40,6 @@ fun <Input : Any, Output : Any> start(
             config = config ?: RuntimeConfig(),
         )
         runtime.setOnOutput(emitter::onNext)
-        runtime.setOnError(emitter::onError)
```

Comment:
There is no need to emit the error as we rely on the global error listener instead.